_posts/2019/2019-01-20-recruitment-final-list-19.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Final recruitment list For 1st year (2019)
+excerpt: Recruitment results 2019.
+---
+
+We interviewed 90 people before releasing the final list of selected people.
+It was a really tough job to filter down from the talented applicants to the
+final list. Sadly, we have limited vacancies and can only take in a limited
+number. But this is not the end of line for others, there's still a way to
+[join SDSLabs](/2014/01/how-to-join-sdslabs). We'll continue taking open
+lectures on variety of topics. Stay connected with us on our Facebook
+[page](http://facebook.com/sdslabs) for the updates regarding this.
+
+## Programmers
+
+* Adrij Shikhar <span style="font-size: 14px;">(_CH_)</span>
```

Comment:
Congratulations everyone :tada: 

nitpick: `<small>` is more semantic

Reply:
@hritvi Make a PR fixing this!